tests/importable_test.py

```diff
@@ -1,26 +1,41 @@
 """Smoke tests related to loading entry points."""
 
+import typing
 from importlib.metadata import entry_points as _discover_entry_points
+from subprocess import check_call as _invoke_command
+from sys import executable as _current_runtime
 
 import pytest
 
 
+EXPECTED_ENTRY_POINTS: list[tuple[str, str, str]] = [
+    ('awx_plugins.credentials', 'aim', 'aim', 'aim_plugin'),
+    ('awx_plugins.credentials', 'conjur', 'conjur', 'conjur_plugin'),
+    ('awx_plugins.credentials', 'hashivault_kv', 'hashivault', 'hashivault_kv_plugin'),
+    ('awx_plugins.credentials', 'hashivault_ssh', 'hashivault', 'hashivault_ssh_plugin'),
+    ('awx_plugins.credentials', 'azure_kv', 'azure_kv', 'azure_keyvault_plugin'),
+    ('awx_plugins.credentials', 'centrify_vault_kv', 'centrify_vault', 'centrify_plugin'),
+    ('awx_plugins.credentials', 'thycotic_dsv', 'dsv', 'dsv_plugin'),
+    ('awx_plugins.credentials', 'thycotic_tss', 'tss', 'tss_plugin'),
+    ('awx_plugins.credentials', 'aws_secretsmanager_credential', 'aws_secretsmanager', 'aws_secretmanager_plugin'),
+]
+
+
 @pytest.mark.parametrize(
-    'entry_points_group',
-    (
-        'awx.credential_plugins',
-        'awx_plugins.credentials',
-    ),
+    'entry_points_group, expected_entry_point_name, expected_entry_point_module, expected_entry_point_func',
```

Comment:
I think this triggers `flake8-pytest` violations, but I understand why you missed them. Still, let's try to avoid introducing more.
```suggestion
    (
        'entry_points_group',
        'expected_entry_point_name',
        'expected_entry_point_module',
        'expected_entry_point_func',
    )
```

In general, I expect that any new code, especially files coming from the skeleton, should never introduce new linting violations.

Reply:
I never really liked that the first param to pytest.parameterize was a string containing a csv. Good to know that a tuple can be passed.

Reply:
Either a tuple of a list. I configured the linter to require tuples for consistency. Marking this thread as resolved.